package-lock.json

```diff
@@ -0,0 +1,5187 @@
+{
```

Comment:
Better! I don't know if it is me, but I have faced problems with `npm`. The package management is not consistent. Some times I don't get all my dependencies installed.

I am wondering now whether this is the reason why you use yarn? :)